ocrd-import

```diff
@@ -144,6 +144,7 @@ for file in $(find -L . -type f -not -name mets.xml -not -name "*.log" | sort);
     set -e
     trap rollback ERR
     page=p${zeros:0:$((4-${#num}))}$num
+	echo "PAGE=$page"
```

Comment:
better use the log fn

Reply:
I think this was just debugging by @stefanCCS, can go without replacement.